Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,136 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * sendToTrashAction event handler, sends location available in request to trash
+         * and makes the application navigate to parent location
+         *
+         * @method _sendContentToTrash
+         * @protected
+         * @param {Object} e event facade of the sendToTrashAction event
+         */
+        _sendContentToTrash: function (e) {
+            var app = this.get('app'),
+                that = this,
+                contentService = this.get('capi').getContentService(),
+                location = this.get('location'),
+                locationId = location.get('id'),
+                contentName = e.content.get('name');
+
+            e.preventDefault();
+            this._set('trackOutsideEvents', false);
+            this.fire('confirmBoxOpen', {
+                config: {
+                    title: "Are you sure you want to send this content to trash?",
+                    confirmHandler: Y.bind(function () {
+                        var tasks, parentLocation, trashPath, endLoadParent, endLoadTrashPath;
+
+                        that._sendContentToTrashNotificationStarted(locationId, contentName);
+
+                        tasks = new Y.Parallel();
+                        endLoadParent = tasks.add();
+                        that._loadParent(location, function (error, parentLocationResponse) {
+                            if (error) {
+                                that._sendContentToTrashNotificationError(locationId, contentName);
+                                that._error("Failed to load parent location of " + locationId);
```

Comment:
you should not fire a notification and call `_error`, it's either one or the other as `_error` will soon also use the notification system. In this context, you should fire the notifiy event and not call `_error`.